### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BeckerFrank**
Add Reporting Issues.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -22,6 +22,24 @@ monitors your programming activity to create a "task context" that focuses your
 relevant artifacts to the task-at-hand. This puts the information you need at your fingertips and improves productivity
 by reducing information overload, facilitating multitasking and easing the sharing of expertise. 
 
-### Developing and Contributing
+## Reporting issues
+
+The Eclipse Mylyn project is split into two Git repositories, which are all part of this organization.
+- https://github.com/eclipse-mylyn/org.eclipse.mylyn.docs
```
Comment (wimjongman) [inline profile/README.md:28]: Make this a link to the issues and not to the repo.

File: profile/README.md [hunk 2]
```diff
@@ -22,6 +22,24 @@ monitors your programming activity to create a "task context" that focuses your
...
+  - Eclipse EPUB Tools
+  - Mylyn HTMLText
+  - Mylyn WikiText
+- https://github.com/eclipse-mylyn/org.eclipse.mylyn
```
Comment (wimjongman) [inline profile/README.md:32]: Make this a link to the issues and not to the repo.

**Review by wimjongman — CHANGES_REQUESTED** [review]

**Comment by wimjongman** [comment]
Thanks, Frank. I suggest to link directly to the issues.

File: profile/README.md (hunk 1 above)
Comment (BeckerFrank) [reply profile/README.md]: changed with commit 3b8092404a0bc3b9d1c38d097e2c9ab9089b9005

File: profile/README.md (hunk 2 above)
Comment (BeckerFrank) [reply profile/README.md]: changed with commit 3b8092404a0bc3b9d1c38d097e2c9ab9089b9005

**Review by wimjongman — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**